### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by kumsy**
@kdious updated README.rst (explanation rewording).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -55,9 +55,9 @@ the ``build_url`` method (mainly helpful for testing):
 Testing
 -------
 
-Tests have been for python 2.7 and python 3 using ``pytest``. The unit
-tests do call the actual PurgoMalum production API in order to test
-against potential API changes.
+Tests have been for python 2.7 and python 3 using ``pytest``. In order
+to test against potential API changes, unit tests call PurgoMalum's
```
Comment (kdious) [inline README.rst]: Change this to: 

```
Tests have been for python 2.7 and python 3 using ``pytest``. The unit
tests do call the actual LibInsult's `production API <https://insult.mattbas.org/api/>`__ in order to test
against potential API changes.
```

Comment (kdious) [reply README.rst]: This should then render "production API" as a hyperlink to the API page.

**Review by kdious — CHANGES_REQUESTED** [review]
See comment.

**Comment by kumsy** [comment]
@kdious documentation with hyperlink to production API added.

**Review by kdious — APPROVED** [review]

**Pull request merged**